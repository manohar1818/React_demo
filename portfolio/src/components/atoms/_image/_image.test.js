import React from 'react';
import { create } from "react-test-renderer";
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Image from './_image';

describe("Image component",() =>{
    test("Matches the snapshot",() => {
        const image = create(<Image />);
        expect(image.toJSON()).toMatchSnapshot();

    })
    test("check whether Image  element is rendered",() => {
        const { getByTestId, getByText } = render(<Image/>);
        expect(getByTestId("image")).toBeInTheDocument();
        
    });
   

});