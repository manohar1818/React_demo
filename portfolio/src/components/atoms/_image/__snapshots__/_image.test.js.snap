// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component Matches the snapshot 1`] = `
<div
  className="makeStyles-styleImage-1"
  data-testid="image"
>
  <div
    className="makeStyles-picture-2"
    style={
      Object {
        "backgroundImage": "URL(undefined)",
      }
    }
  />
</div>
`;
